test/component/deceased/testAlias.js

```diff
@@ -1,16 +1,19 @@
 const TestWrapper = require('test/util/TestWrapper'),
+    {set} = require('lodash'),
     DeceasedOtherNames = require('app/steps/ui/deceased/otherNames/index'),
     DeceasedMarried = require('app/steps/ui/deceased/married/index'),
     DeceasedDod = require('app/steps/ui/deceased/dod/index');
 
+
 describe('deceased-alias', () => {
-    let testWrapper;
+    let testWrapper, sessionData;
     const expectedNextUrlForDeceasedOtherNames = DeceasedOtherNames.getUrl();
     const expectedNextUrlForDeceasedMarried = DeceasedMarried.getUrl();
     const expectedNextUrlForDeceasedDod = DeceasedDod.getUrl();
 
     beforeEach(() => {
         testWrapper = new TestWrapper('DeceasedAlias');
+        sessionData = {};
```

Comment:
We don't need to set `sessionData` here as it can be set in the test that uses it.